src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -556,7 +578,7 @@ export default {
 			this.quickFiltersOptions.gender = [
 				{
 					title: 'All genders',
-					key: '',
+					key: 'all',
```

Comment:
When did this key change? It used to mean that empty meant all values.

Reply:
oops forgot to change this back from testing